diff --git a/src/background/vvppManager.ts b/src/background/vvppManager.ts
index abd2e3b..e093221 100644
--- a/src/background/vvppManager.ts
+++ b/src/background/vvppManager.ts
@@ -201,10 +201,7 @@ export class VvppManager {
           throw new Error("7z path is not defined");
         }
         if (import.meta.env.PROD) {
-          sevenZipPath = path.join(
-            path.dirname(app.getPath("exe")),
-            sevenZipPath
-          );
+          sevenZipPath = path.join(import.meta.env.VITE_APP_BASE, sevenZipPath);
         }
         log.log(
           "Spawning 7z:",
